server/controllers/session.ctrl.js

```diff
@@ -0,0 +1,29 @@
+const axios = require('axios');
+
+const client = axios.create({
+  baseURL: 'http://localhost:8000/'
+});
+
+module.exports = {
+  createSession: (req, res, next) => {
+    console.log(req.body.code);
```

Comment:
You're keeping console.logs?, you should look into a logger like Bunyan. I have a rule set on eslint to not allow console outs. I usually use console for debugging.

Reply:
It was something I used for local development, it should be removed. I'm probably not going to log it to a file either.

Reply:
the logger doesn't have to log into a file. It can log to just stdout.

Reply:
Yeah it probably shouldn't do that either given the security issue of having an auth code displayed.